site/content/en/concepts/packaging/_index.md

```diff
@@ -25,4 +25,54 @@ For compatibility with other arbitrary formats, kpt supports generating
 resource configuration data from templates, configuration DSLs, and programs
 using [source functions].
 
+## Nested Packages
+
+A `nested package` is a `kpt` package which contains 1 or more `kpt` packages
+in its directory tree.
+
+Here is an example directory structure of a `nested package` with name `foo`
+
+```sh
+foo # nested package
```

Comment:
So I'm thinking thinking that this is not a nested package, but rather the top-level package. I'm thinking nested package and subpackage means the same.

Reply:
There are two dimensions.

1. A package may or may not contain other packages.
2. A subpackage may be declared using `Kptfile.dependencies` or by virtue of containing a subpackage dir.

For #1, I think we just need to define the term 'subpackage' and avoid 'composite' or 'nested', etc.

Not sure what we want to call #2, do we have terms for them? `dependencies` is going to be overloaded with eventual plans around declarative Pipelines.

Reply:
Removed all the terms `composite` or `nested`, etc. Renamed the concept to `subpackages` and explained it. Guides are also updated accordingly. `Kptfile.dependencies` is not documented officially yet, so restricted the discussion to packages in the directory tree of another package. We can expand more once we ship `pkg` features. PTAL.